extensions/src/ACRE2Mumble/mumble_includes/MumbleAPI_v_1_0_x.h

```diff
@@ -17,9 +17,9 @@
 #define MUMBLE_PLUGIN_API_MINOR_MACRO 0
 #define MUMBLE_PLUGIN_API_PATCH_MACRO 0
 
-constexpr int32_t MUMBLE_PLUGIN_API_MAJOR        = MUMBLE_PLUGIN_API_MAJOR_MACRO;
-constexpr int32_t MUMBLE_PLUGIN_API_MINOR        = MUMBLE_PLUGIN_API_MINOR_MACRO;
-constexpr int32_t MUMBLE_PLUGIN_API_PATCH        = MUMBLE_PLUGIN_API_PATCH_MACRO;
+const int32_t MUMBLE_PLUGIN_API_MAJOR            = MUMBLE_PLUGIN_API_MAJOR_MACRO;
```

Comment:
why do you remove the constexpr?

Reply:
Because the API is supposed to the C-compatible but `constexpr` is cpp-only